test/unit/src/audio/FftUnit.cpp

```diff
@@ -0,0 +1,60 @@
+#include "cinder/Cinder.h"
+
+// FIXME: OOURA roundtrip FFT seems to be broken on windows for sizeFft = 4 (https://github.com/cinder/Cinder/issues/1263)
+#if ! defined( CINDER_MSW )
+
+#include "catch.hpp"
+#include "utils.h"
+
+#include "cinder/Log.h"
+#include "cinder/audio/dsp/Fft.h"
+
+#include <iostream>
+
+using namespace ci::audio;
+
+namespace {
+
+void computeRoundTrip( size_t sizeFft )
+{
+	dsp::Fft fft( sizeFft );

```

Comment:
Or just `CAPTURE( sizeFft );` and it will print `sizeFft := 4` if an assertion fails.